.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -33,6 +33,9 @@ recommend you upload it.
 
 Go to the command palette (Cmd+Shift+P on Mac, Ctrl+Shift+P on Windows), search `Open Angular Server log`
 
+Set Angular Log level to verbose.
+Open settings (File/Preferences/Settings on Windows, Code/Preferences/Settings on Mac), go to Extensions/Angular Language Service, set `Angular: Log` to `verbose`.
```

Comment:
```suggestion
Open settings (File/Preferences/Settings on Windows, Code/Preferences/Settings on Mac), go to Extensions/Angular Language Service, set `angular.log` to `verbose`.
```

Reply:
The actual config name is `angular.log`:
https://github.com/angular/vscode-ng-language-service/blob/d8666d8352e09febecdb98e16b8738c64264a707/package.json#L74-L84